documentation/topics/glossary.md

```diff
@@ -0,0 +1,95 @@
+# Glossary
+
+(This is a work in progress and will be expanded over time!)
+
+## Action
+
+An action describes an operation that can be performed for a given resource; it is the _verb_ to a resource's _noun_. Examples of actions:
+
+- User._create_
+- Comment._delete_
+- BlogPost._publish_
+- Article._search_
+
+Ash supports four different types of actions - `create`, `read`, `update` and `destroy` (collectively often abbreviated as CRUD). A resource can define multiple actions per action type, eg. a _publish_ action would be considered an `update` because it is updating an existing instance of a resource.
+
+See the [Actions guide](/documentation/topics/actions.md) for more information.
+
+## Actor
+
+The entity that performs an action. Most actions are run on direct user request, eg. if a user presses a Create button on a page then the actor is the user; but an actor might also be an organization, a group, or the system itself.
+
+Actors are referenced during authorization - ensuring that the actor is allowed to perform an action, before it takes place.
```

Comment:
The actor can also be used in the action to do things like relate the current user to the record as an "owner".